roles/katello_pulp3/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: create static dir
+  file:
+    path: '/var/lib/pulp/static/'
+    state: directory
+    mode: '777'
+
+- include_role:
+    name: "{{ item }}"
+  with_items:
+    - pulp-database
+    - pulp-workers
+    - pulp-resource-manager
+    - pulp-content
+  vars:
+    pulp_content_host: "{{ ansible_fqdn }}"
+    pulp_default_admin_password: password
+    pulp_requirements_dir: "https://raw.githubusercontent.com/pulp/pulpcore/master/"
+    pulp_plugin_source_dir: "git+https://github.com/pulp/pulpcore-plugin.git#egg=pulpcore-plugin"
+    pulp_source_dir: "git+https://github.com/pulp/pulpcore.git#egg=pulpcore"
+    pulp_install_plugins:
+      pulp-file:
+        app_label: "file"
+        source_dir: "git+https://github.com/pulp/pulp_file.git#egg=pulp-file"
+      pulp-docker:
+        app_label: "docker"
+        source_dir: "git+https://github.com/pulp/pulp_docker.git#egg=pulp-docker"
+      pulp-ansible:
+        app_label: "ansible"
+        source_dir: "git+https://github.com/pulp/pulp_ansible.git#egg=pulp-ansible"
```

Comment:
What about setting all of these in defaults like some are currently set?

Reply:
Moving this to defaults didn't actually work, i believe due to https://github.com/ansible/ansible/issues/47023